types/markdown-it-emoji/index.d.ts

```diff
@@ -3,16 +3,43 @@
 // Definitions by: Christopher Quadflieg <https://github.com/Shinigami92>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-import { PluginSimple, PluginWithOptions } from 'markdown-it/lib';
+import { PluginSimple, PluginWithOptions } from "markdown-it/lib";
 
 export as namespace markdownitEmoji;
 
 declare namespace markdownitEmoji {
     interface Options {
         defs?: Record<string, string>;
         enabled?: string[];
-        shortcuts?: Record<string, string | string[]>;
+        shortcuts?: Record<Shortcut, string | string[]>;
```

Comment:
**suggestion:** What about using [LiteralUnion<Shortcut>](https://github.com/microsoft/TypeScript/issues/29729#issuecomment-471566609)?

Reply:
can you be more specific? I was thinking shortcuts are closed list here, thx!

Reply:
`LiteralUnion` is just a future proof version of an union type so you get auto-completion but also are allowed to insert any other string if needed. So if there are new shortcuts, they can already be used without updating these type. Where if `LiteralUnion` is not used, you getting a compile-time error and need workarounds like `// @ts-expect-error` or `as unknown as Shortcut`

Reply:
I've amended the code, the 'Shortcut' type itself is now literal union. But I cannot recall how this
is supposed to work as keyed value for `shortcuts` properties (also amended in the update to represents
optional keys for shortcuts)